feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.component.html

```diff
@@ -3,17 +3,27 @@
 >
   <div class="cx-menu-bar" role="tablist">
     <button
+      #menuTab
+      [tabindex]="getTabIndexForMenuTab()"
       role="tab"
       [attr.aria-selected]="!showFilter"
       [class.active]="!showFilter"
+      (keydown)="switchTabOnArrowPress($event, '#menuTab')"
+      (keydown.enter)="onMenu()"
+      (keydown.space)="onMenu()"
```

Comment:
any reason you did this change without adding a component test for it?

Reply:
Done.